serialzy/version/version

```diff
@@ -1 +1 @@
-1.2.1
+1.2.2
```

Comment:
It doesn't matter relieve, but I believe your edits deserves an minor version bump, not micro :)